presto-main/src/main/java/com/facebook/presto/testing/MaterializedResult.java

```diff
@@ -424,6 +425,51 @@ public static MaterializedResult materializeSourceDataStream(ConnectorSession se
         return builder.build();
     }
 
+    public static boolean resultsEqual(MaterializedResult viewTable, MaterializedResult baseTable)
```

Comment:
Why do we need an extra function instead of directly call `assertEquals(viewTable, baseTable)`? The `equals` function of `MaterializedResult` should just work.

Reply:
Good point. I've replaced calling the function with `assertEquals(viewTable, baseTable)`.